shell/src/main/java/org/apache/accumulo/shell/commands/ConfigCommand.java

```diff
@@ -187,6 +205,10 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       try {
         systemConfig
             .putAll(shellState.getAccumuloClient().instanceOperations().getSystemConfiguration());
+        Shell.log.warn("System configuration is dependent on the server's site configuration and"
```

Comment:
Should only print this once per shell session. It's too verbose.

Reply:
Moved this to a ConfigCommand constructor in 5fe3f9e so that it's only printed once on Shell startup.